doc/TUTORIAL.md

```diff
@@ -152,10 +152,9 @@ The process of turning this object into an image is handled automatically by the
 To try out this example badge:
 
 1. Copy and paste this code into a new file in `/services/example/example.service.js`
-2. Quit the running server with `Control+C`.
-3. Start the server again.
-   `npm start`
-4. Visit the badge at <http://localhost:8080/example/foo.svg>.
+2. The server should restart on its own. (If it doesn't for some reason, quit
```

Comment:
👍 I like the auto-reload call out here. I feel like we should also call out that behavior elsewhere in other docs (so folks know they don't need to manually restart on changes). 

I was hoping to propose some specific suggestions but can't get the wording right.

Reply:
Could probably add it in the readme in the spot I updated!

Though FWIW the console is pretty chatty so it’s pretty easy to notice!

Reply:
Good shout. I've added a few sentences in the readme.